dissect/target/plugins/os/unix/log/journal.py

```diff
@@ -314,7 +313,7 @@ def decode_value(self, value: bytes) -> tuple[str, str]:
 
         return key, value
 
-    def __iter__(self) -> Iterator[Instance]:
+    def __iter__(self) -> Iterator[dict[str, Any]]:
```

Comment:
```suggestion
    def __iter__(self) -> Iterator[dict[str, int | str]]:
```

Aren't the dict values in `event` not always either the ouput of `tx.from_unix_ts()` which is `int`, or the output of `self.decode_value()` which is `str`?